src/ode_solver/reduced_order_ode_solver.h

```diff
@@ -0,0 +1,50 @@
+#ifndef __REDUCED_ORDER_ODE_SOLVER__
+#define __REDUCED_ORDER_ODE_SOLVER__
+
+#include "dg/dg.h"
+#include "ode_solver_base.h"
+#include "reduced_order/pod_basis_base.h"
+
+namespace PHiLiP {
+namespace ODE {
+
+/// POD-Petrov-Galerkin ODE solver derived from ODESolver.
+#if PHILIP_DIM==1
+template <int dim, typename real, typename MeshType = dealii::Triangulation<dim>>
+#else
+template <int dim, typename real, typename MeshType = dealii::parallel::distributed::Triangulation<dim>>
+#endif
+class ReducedOrderODESolver: public ODESolverBase<dim, real, MeshType>
+{
+protected:
+    /// Default constructor that will set the constants.
+    ReducedOrderODESolver(std::shared_ptr< DGBase<dim, real, MeshType> > dg_input, std::shared_ptr<ProperOrthogonalDecomposition::PODBase<dim>> pod); ///< Constructor.
+
+public:
+    ///POD
+    std::shared_ptr<ProperOrthogonalDecomposition::PODBase<dim>> pod;
+
+    /// Destructor
+    ~ReducedOrderODESolver() {};
```

Comment:
Destructor can probably be virtual here.

Reply:
Fixed in most recent commit.